genesis.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gossamer",
+  "id": "gossamer",
+  "bootNodes": [
+    "/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj"
+  ],
+  "protocolId": "gossamer",
+  "genesis": {
+    "raw": [
```

Comment:
What is genesis raw exactly? Is it the initial state?

What about things like the initial randomness, initial set of validators, etc? We don't need them for this PR, but perhaps we should compile a list of things we need to add.

Reply:
yeah it's raw data to save into the storage trie, it's key-value pairs.

I believe those things will also go in here, I'm just not sure about the format yet :) so yes let's add it to a doc to consider later